frontend/src/app/modules/auth/register-form/register-form.component.scss

```diff
@@ -56,6 +56,9 @@
         }
         button:disabled {
             cursor: default;
+            background: #e0d8d7;
```

Comment:
For me, it still looks like enabled
![image](https://user-images.githubusercontent.com/20342130/154928860-5dd73e52-d041-44d4-8f98-a2e23ee3cd7f.png)

Reply:
I have testeted in Firefox, Chrome, Opera, and it's forks for me with this test data.